gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinTaskRunner.java

```diff
@@ -113,6 +107,9 @@
  */
 @Alpha
 public class GobblinTaskRunner implements StandardMetricsBridge {
+  // Working directory key for applications. This config is set dynamically.
+  public static final String CLUSTER_APP_WORK_DIR = GobblinClusterConfigurationKeys.GOBBLIN_CLUSTER_PREFIX + "appWorkDir";
```

Comment:
If set dynamically, shall make it package private instead of globally accessible ?

Reply:
Making it package-private makes the config inaccessible inside IntegrationBasicSuite. Hence, set it to public.